### DISCUSSION TITLE
How to test the "echo" examples?

### DISCUSSION THREAD

**Discussion by raindust** [Totodore/socketioxide#197] [Q&A] [answered]
Take https://github.com/Totodore/socketioxide/tree/main/examples/axum-echo for example, when I cd into this directory and run with "cargo run". Other use browsers (I tested Firefox and Chrome) or directly use curl in the terminal, it only returns "Hello, World!" and no more.
And line https://github.com/Totodore/socketioxide/blob/main/examples/axum-echo/axum_echo.rs#L12 never prints on the server side, is there anything I missed?

**Comment by tausifcreates** [comment, marked as answer]
@raindust 
Hi! The 'Hello World' line prints on a Get request from browser or curl.
The socket never connects, probably because you are not using a socket io client.
You can get the browser client here: https://socket.io/docs/v4/client-api/

**Reply by raindust** [reply]
I see, thank you
